business_/good_service.py

```diff
@@ -0,0 +1,39 @@
+from good_repository import GoodRepository
+from good_entity import GoodEntity
+from good_dto import GoodDto
+
+
+class GoodService:
+    def __init__(self, good_repo: GoodRepository):
+        self._good_rep = good_repo
+
+    def get(self, good_id: int) -> GoodEntity:
+        good_dataclass = self._good_rep.read_one(good_id)
+        return good_dataclass
+
+    def delete(self, good_id: int) -> GoodEntity:
+        good_dataclass = self._good_rep.delete_one(good_id)
+        return good_dataclass
+
+    def create(self, good_dto: GoodDto) -> GoodEntity:
```

Comment:
```suggestion
    def create(self, good_dto: GoodCreateDto) -> GoodEntity:
```

Для create и update должны быть разные dto